### PR TITLE
Dropdown rem to pixel

### PR DESCRIPTION
**Pull request by matzimowski**
there was an issue on some viewports

before:
screen shot 2015-09-22 at 12 25 30

after:
screen shot 2015-09-22 at 12 25 46

### CODE REVIEW

File: src/sass/_mixins.scss [hunk 1]
```diff
@@ -119,3 +119,11 @@
 @function gutter($size){
   @return $size * $layoutDefaultPadding;
 }
+
+@function strip-units($number) {
+  @return $number / ($number * 0 + 1);
```
Comment (vergilius) [inline src/sass/_mixins.scss]: `($number * 0 + 1) == 1` ?

Comment (matzimowski) [inline src/sass/_mixins.scss]: https://github.com/sass/sass/issues/533

Comment (vergilius) [inline src/sass/_mixins.scss]: :+1: add please comment in the code

Comment (kdzwinel) [inline src/sass/_mixins.scss]: https://css-tricks.com/snippets/sass/strip-unit-function/ - don't you need these ifs there?

**Comment by kdzwinel** [comment]
Why exactly are we doing this? Value calculated with rems wasn't precise enough? Don't we have some other places where this should be done?

**Comment by matzimowski** [comment]
There is a problem connected with borders, border-size has to be an integer.
so this strange situation happens when border was 1.992px (but transformed to 2px) and left was like 1.992px and remained like this.
I found this issue only here so far.

File: src/sass/_mixins.scss (hunk 1 above)
Comment (matzimowski) [inline src/sass/_mixins.scss]: we don't care, we do not have to  generalise it yet :older_man:

**Comment by kdzwinel** [comment]
:+1:

File: src/sass/_mixins.scss (hunk 1 above)
Comment (matzimowski) [inline src/sass/_mixins.scss]: C'mon, we will update the function according to ur comment.

**Comment by aju** [comment]
:ok:

**Pull request merged**